snuba/state/__init__.py

```diff
@@ -88,26 +87,6 @@ class MismatchedTypeException(Exception):
     new_type: Type[Any]
 
 
-def get_concurrent(bucket: str) -> Any:
```

Comment:
These functions are used by the web dashboard, I don't think you can just remove them.